package.json

```diff
@@ -59,6 +59,9 @@
     "setupFiles": [
       "<rootDir>/setup.ts"
     ],
+    "reporters": [
```

Comment:
This is a custom reporter which hides console.log's for passed tests. It makes the tests output more readable.